resources/test/normalizers/configfile/en

```diff
@@ -3,5 +3,5 @@ LocalizedFile config de ./resources/test/normalizers/configfile
 
 Lowercase
 
-# And re-use regexreplace for en
-localizedfile RegexReplace it ./resources/test/normalizers/regexreplace
\ No newline at end of file
+# And re-use regex for en
+localizedfile Regex it ./resources/test/normalizers/regex
```

Comment:
What's this 'localizedfile' about? It's only mentioned in this config file, which I don't think is being used by any tests. Same goes with the `de` config file. It might be worth deleting these files

Reply:
(removed files, since no longer relevant)